python/lib/dependabot/python/file_parser.rb

```diff
@@ -104,6 +106,38 @@ def requirement_dependencies
         dependencies
       end
 
+      def instrument_package_manager_version
+        package_managers = {}
+        package_managers["poetry"] = poetry_version if using_poetry?
+        package_managers["pipenv"] = pipenv_version if pipfile
+        package_managers["setup_py"] = setup_py_version if setup_file
+        package_managers["pip"] = pip_version if requirement_files.any?
+
+        Dependabot.instrument(
+          "dependabot.file_parser.package_manager_version_parsed",
+          ecosystem: "python",
+          package_managers: package_managers
+        )
+      end
+
+      # TODO: get actual versions
+
+      def poetry_version
+        "unknown"
+      end
+
+      def pipenv_version
+        "unknown"
+      end
+
+      def pip_version
+        "unknown"
+      end
+
+      def setup_py_version
+        "unknown"
```

Comment:
I'm not sure if we can even get a version for this?

Reply:
yeah not sure there is a version?

Reply:
but this would be valuable even without version information, just knowing which ones are actually used